### DISCUSSION TITLE
Strange padding in treemap chart

### DISCUSSION THREAD

**Discussion by zdoryk** [apexcharts/apexcharts.js#3561] [General]
### Hi
I am using nuxt.js but i think that the problem is in library:
In TreeMap chart there is strange offset or padding i don't know how to name it right. 
Even on demo page on offical site there is offset for toolbar and title in svg element:

![image](https://user-images.githubusercontent.com/63752476/208499452-40c0cf36-6cc9-4d54-9804-94a00d8dce35.png)

But in my situation i have toolbar show false and even don't have title at all:
![image](https://user-images.githubusercontent.com/63752476/208499538-466883b3-4eb4-4e5c-9f8e-f6784d1695ac.png)

I've set ```chart: {toolbar: {show: false}}```

I was trying to do: https://github.com/apexcharts/apexcharts.js/issues/178#issuecomment-435580300

What can i do to clear these padding around treemap?

Thanks and best regards.

**Comment by chris-oleson** [comment]
Also having this issue; would love a way to remove the unnecessary padding

**Comment by sorenstaun** [comment]
I'm in line here looking for a solution.

**Comment by MizuBishi** [comment]
+1

**Comment by rodneymandap** [comment]
Having the same issue as well. +1

**Comment by Alext162** [comment]
Same issue!

**Comment by T0573** [comment]
Same issue, did anyone find a solution / workaround?
